### PR TITLE
feature/ve-evs-prod

### PR DESCRIPTION
**Pull request by ShelleyMelchior-NOAA**
## Pull Request Testing ##

- [x ] Describe testing already performed for this Pull Request:</br>

**All three steps (prep, stats, and plots) have been tested across global_det and global_ens.**

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

**I recommend a quick spot check on another component or two, but not to delay much. The best test will be getting the modulefiles updates into the EVS para for the full on confirmation test.**

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by PerryShafran-NOAA** [comment]
I cloned the branch and am running a few quickie tests on a few scripts.

Perry

File: modulefiles/nfcens/nfcens_prep.sh
Comment (malloryprow) [inline modulefiles/nfcens/nfcens_prep.sh:1]: module use /apps/prod/lmodules/ needs to be updated to include intel/${intel_ver}

(module use /apps/prod/lmodules/intel/${intel_ver})

Comment (PerryShafran-NOAA) [reply modulefiles/nfcens/nfcens_prep.sh:1]: Shelley did this so it's good.

Perry

**Comment by PerryShafran-NOAA** [comment]
I did a quickie test of one component; finished in normal time.  No further tests needed; will do further monitoring in the parallel.

Ready to merge.

Perry

**Review by PerryShafran-NOAA — APPROVED** [review]

**Pull request merged**